assets/svg/tag.svg

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="utf-8"?>
+<svg version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" width="32" height="32" viewBox="0 0 32 32">
```

Comment:
`xmlns:xlink="http://www.w3.org/1999/xlink"` this svg extension is not used in this file we can delete it